org.metaborg.spoofax.shell.core/src/main/java/org/metaborg/spoofax/shell/services/ServiceUnavailableException.java

```diff
@@ -0,0 +1,17 @@
+package org.metaborg.spoofax.shell.services;
+
+public class ServiceUnavailableException extends Exception {
```

Comment:
It is not a `MetaBorg` exception (that is the Spoofax logic). Could create a super type for Shell exceptions, but currently the existing `CommandNotFoundException` simply extends from `Exception` as well.
Only benefit would be to catch Shell exceptions in one nominator.